sos/plugins/abrt.py

```diff
@@ -24,16 +24,15 @@ class Abrt(Plugin, RedHatPlugin):
         ("detailed", 'collect detailed info for every report', 'slow', False)
     ]
 
-    def info_detailed(self, list_file):
-        for line in open(list_file).read().splitlines():
-            if line.startswith("Directory:"):
-                self.add_cmd_output("abrt-cli info -d '%s'" % line.split()[1])
-
     def setup(self):
         self.add_cmd_output("abrt-cli status")
-        list_file = self.get_cmd_output_now("abrt-cli list")
-        if self.get_option('detailed') and list_file:
-            self.info_detailed(list_file)
+        abrt_list = self.get_cmd_output_now("abrt-cli list")
+        if self.get_option("detailed") and abrt_list:
```

Comment:
Technically, abrt_list (as a return value of `get_cmd_output_now`) will always be a dict, so `and abrt_list` will always pass. We have few options here:

- test for `abrt_list["status"] == 0`
- test for `abrt_list["output"]` - that is bit redundant due to the further iteration over output splitlines
- skip this test

I am inclined to skip the test.

Reply:
Yep, oversight on my part - should be `abrt_list['status'] == 0` here.